runner/ansible/roles/2.1.2/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.2'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # SAPHanaTopology is not configured, then pass
+    cibadmin -Q --xpath "//primitive[@type='SAPHanaTopology']/@type" || exit 0
+    # if configured and not clone, then fail
+    cibadmin -Q --xpath "//clone/primitive[@type='SAPHanaTopology']/@type" || exit 1
+    # check the following values, if any do not match expected fail.
+    [[ $(cibadmin -Q --xpath "//primitive[@type='SAPHanaTopology']/operations/op [@name='monitor']" | grep -oP 'interval="\K[^"]+') != "{{ expected[test_id + '.monitor_interval'] }}" ]] && exit 1
```

Comment:
`//primitive[@type='SAPHanaTopology']/operations/op` is repeated many times. We could put it in a variable to reduce the amount of text

Reply:
Agreed, but I can't make that change from this system. I played with that when doing these and it is a bit tricky. And I can't test it until I get back home. I am cool with your making this change though.